src/index.js

```diff
@@ -144,25 +149,44 @@ exports.IPFS = P2P
  * Validation funcs
```

Comment:
Can you update the above exports like in #78 ?

Reply:
Done